### DISCUSSION TITLE
matc warnings, cause for concern?

### DISCUSSION THREAD

**Discussion by jwwalker** [google/filament#4072] [Q&A] [answered]
When I use `matc`, I tend to get a lot of warnings, like:

    WARNING: :0:0:0: The function '%74 = OpFunction %7 None %73' could not be inlined because the return instruction is not at the end of the function. This could be fixed by running merge-return before inlining.
    WARNING: :0:0:0: The function '%154 = OpFunction %7 None %105' could not be inlined because the return instruction is not at the end of the function. This could be fixed by running merge-return before inlining.

Is this anything I should worry about?  (Platform: macOS)

**Comment by romainguy** [comment, marked as answer]
No nothing to worry about, those are just optimization failures (you can turn off optimizations if you wish).
